src/main.rs

```diff
@@ -39,40 +31,36 @@ fn main() {
 
     if requested_version == py::RequestedVersion::Any {
         if let Some(venv_root) = env::var_os("VIRTUAL_ENV") {
-            let mut path = path::PathBuf::new();
+            let mut path = PathBuf::new();
             path.push(venv_root);
             path.push("bin");
             path.push("python");
             // TODO: is_file() check?
-            match run(&path, &args) {
-                Err(e) => println!("{:?}", e),
-                Ok(_) => (),
-            };
+            if let Err(e) = Command::new(path).args(args).status() {
```

Comment:
So I actually purposefully used `execv()` because I wanted to substitute the running process instead of ending up with two processes. So unless `Command` can do that instead of spawning a second process then I want to stick with the current solution of using `execv()` (somehow; I'm not attached to `unistd` specifically).